scripts/superdesk/editor2/editor.js

```diff
@@ -1334,26 +1334,47 @@ angular.module('superdesk.editor2', [
         };
     }])
     .run(['embedService', 'iframelyService', function(embedService, iframelyService) {
-        var pattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
-        var loader = '//snappa.embed.pressassociation.io/playbuzz.js';
-        var html = [
+        var playBuzzPattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
+        var playBuzzlLoader = '//snappa.embed.pressassociation.io/playbuzz.js';
```

Comment:
Ok I will do it in another pr  (this is out of this scope) .  But in
general we can't maintain in all the embed scripts that we support. It
would be an endless job. We must rely on external sources as much as
possible.

On Mon, May 2, 2016, 13:08 Yauhen Kirylau <notifications@github.com> wrote:

> In scripts/superdesk/editor2/editor.js
> <https://github.com/superdesk/superdesk-client-core/pull/388#discussion_r61726657>
> :
>
> > @@ -1334,26 +1334,47 @@ angular.module('superdesk.editor2', [
> >          };
> >      }])
> >      .run(['embedService', 'iframelyService', function(embedService, iframelyService) {
> > -        var pattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
> > -        var loader = '//snappa.embed.pressassociation.io/playbuzz.js';
> > -        var html = [
> > +        var playBuzzPattern = 'https?:\/\/(?:www)\.playbuzz\.com(.*)$';
> > +        var playBuzzlLoader = '//snappa.embed.pressassociation.io/playbuzz.js';
>
> the fact what that script could be updated independently from code
> actually sounds more like a con to me
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk-client-core/pull/388/files/784ea27468c4de5cab49936b28a1b6c6daa0733b#r61726657>
>